functions/auth.js

```diff
@@ -10,3 +10,12 @@ exports.checkAuth = (context) => {
     throw new functions.https.HttpsError("permission-denied", "Token invalid");
   }
 };
+
+// Case-insensitive first character for iDevices
+const lowercaseFirstChar = (password) =>
+  password.charAt(0).toLowerCase() + password.substring(1);
```

Comment:
@thecadams What's the context of this being needed? The comment says for iDevices, but it doesn't provide any reference/reasoning why.

Reply:
On some apple devices when typing in an input the first letter is capital by default. I believe this was added to make the password input more friendly when it come to UX.

Reply:
I would expect that that isn't going to be the case in a password field though? Is it?

I guess this is fine to keep for now. Though it speaks to a potentially bigger issue too, which is storing passwords in plaintext generally isn't considered good practice. It might be fine in this case since we may not consider this feature a 'high security' location for it to exist. But yeah.. just feels wrong all round.

Reply:
> I would expect that that isn't going to be the case in a password field though? Is it?

Now that you mention it, it might be.